src/main/scala-2.12/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -29,18 +29,18 @@ object JmapMessages {
   type JmapParameters = Map[String, Any]
   val NO_PARAMETERS : JmapParameters = Map()
 
-  def sendMessages(messageId: MessageId, subject: Subject, textBody: TextBody) =
+  def sendMessages() =
     JmapAuthentication.authenticatedQuery("sendMessages", "/jmap")
       .body(StringBody(
         s"""[[
           "setMessages",
           {
             "create": {
-              "${messageId.id}" : {
+              "$${messageId}" : {
```

Comment:
idem you could remove the scala string interpolation and simplify all this string